src/sipsak.c

```diff
@@ -233,13 +233,13 @@ void print_help() {
 		"  -P NUMBER         Number of processes to start\n"
 		"  -A NUMBER         number of test runs and print just timings\n"
 		"  -S                use same port for receiving and sending\n"
-		"  -c SIPURI         use the given uri as From in MESSAGE\n"
+		"  -c SIPURI         use the given URI as From in MESSAGE\n"
 		"  -D NUMBER         timeout multiplier for INVITE transactions\n"
 		"                    on non-reliable transports (default: 64)\n"
 		"  -Z NUMBER         timeout T1 in ms (default: %i)\n"
 		"  -E STRING         specify transport to be used\n"
 		"  -j STRING         adds additional headers to the request\n"
-		"  -J STRING         ha1 hash for authentication instead of password\n"
+		"  -J STRING         sha1 hash for authentication instead of password\n"
```

Comment:
This actually refers to HA1 which is one part of how the password challenges are calculated in SIP. So this is the only not typo in your patch :-)
Could you maybe change it to HA1 to make it more clear that defined term?

Reply:
Oh, I see, I guess I mixed things up. :smile:

Will push a fix shortly.

Cheers